force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -16,13 +16,28 @@
  * 
  * Optional Challenge: Use a trigger handler class to implement the trigger logic.
  */
-trigger ContactTrigger on Contact(before insert) {
-	// When a contact is inserted
-	// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
+trigger ContactTrigger on Contact(before insert, before update) {
+	
+	if(Trigger.isInsert){
+		for(Contact cont : Trigger.new) {
+			// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
+			if(cont.DummyJSON_Id__c == null) {
+				Integer randomNumber = Integer.valueOf((Math.random() * 100));
+				cont.DummyJSON_Id__c = String.valueOf(randomNumber);
+			}
+			// if DummyJSON_Id__c is less than or equal to 100, call the getDummyJSONUserFromId API
+			else if(Integer.valueOf(cont.DummyJSON_Id__c) <= 100){
+				ContactTriggerHandler.makeInsertCallout(cont.DummyJSON_Id__c);
```

Comment:
Try to make your triggers bulkify by sending a list of 200 records to the TriggerHandler
Best to send the trigger.new() to the handler